backendAS/pg.sql

```diff
@@ -0,0 +1,35 @@
+-- TEST
+DROP TABLE if exists testsong;
+DROP TABLE if exists testcomment;
+
+CREATE TABLE testsong(
+  id int,
```

Comment:
missing a `primary key` identifier in the table to prevent duplicate records

Reply:
(Noticed this in all of your table schemas)

Reply:
Tables also in need of validators

Reply:
Resolved.

Reply:
Added validators (had done so in the cli previously)